src/Chainweb/Pact/PactService/Checkpointer.hs

```diff
@@ -349,7 +349,12 @@ rewindTo rewindLimit (Just (ParentHeader parent)) = do
         -- This does a restore, i.e. it rewinds the checkpointer back in
         -- history, if needed.
         withCheckpointerWithoutRewind (Just target) "fastForward" $ \pdbenv -> do
-            payload <- liftIO (payloadWithOutputsToPayloadData <$> casLookupM payloadDb bpHash)
+            payload <- liftIO $ casLookup payloadDb bpHash >>= \case
```

Comment:
With this PR `Chainweb.Test.Pact.RemotePactTest/remote pact tests/await network` to hang forever. I think, this is the only change that could explain that behavior.

Before this was throwing a low-level exception from the `Chainweb-storage` package. Now it is throwing an `InternalPactError`.